CyDelete/Resources/CyDelete.plist

```diff
@@ -68,7 +68,7 @@
 		</dict>
 		<dict>
```

Comment:
You're missing entries here to show your donation links for Bitcoin and AliPay :)